packages/main/src/RPA/Tables.py

```diff
@@ -1466,14 +1492,14 @@ def table_head(
         :param table:   Table to read from
         :param count:   Number of lines to read
         :param as_list: Return list instead of Table
+        :return:        Return Table object or List of the selected rows
 
-        Examples:
+        Example:
 
         .. code-block:: robotframework
 
             # Get the first 10 employees
-            ${employees}=    Read worksheet as table    employees.xlsx
-            ${first}=        Table head    ${employees}    10
+            ${first}=    Table head    ${employees}    10
```

Comment:
Is it ok to hide the source of `${employees}`? Seen the same idiom at this [line](https://github.com/robocorp/rpaframework/pull/492/files#diff-25a3bbc35959efc76efba5321b5512ceb20cd1cc4f8896beeaf4b333480a8eb1L1455) as well.

Not a big deal, just curious if it makes more sense in the docs to focus on the keyword effect itself. (rather than including its setup as well)

Reply:
I'm hiding this detail as our experienced users understand where this comes from and my experience with new users is that they blindly copy and paste the examples and think that this is the only way for this to work. To avoid that I wanted to keep our examples more generic and let them determine how to bring in the table.